### PR TITLE
feat(readme): Add better readme

### PR DESCRIPTION
**Pull request by yigiterdev**

### CODE REVIEW

File: CONTRIBUTING.md
Comment (mucahit) [inline CONTRIBUTING.md:7]:
```suggestion
- Fork the repo and create your branch from `next-release`.
```

File: CONTRIBUTING.md
Comment (mucahit) [inline CONTRIBUTING.md:10]:
```suggestion
We use npm to manage installation of dependencies and running various scripts. To get everything installed, make sure you have npm and run `npm install` from the root of the repository.
```

File: CONTRIBUTING.md
Comment (mucahit) [inline CONTRIBUTING.md:73]:
```suggestion
After doing this, you can run `npm run dev` and in this way, you can see the changes you have made to the package simultaneously.
```

File: CONTRIBUTING.md
Comment (mucahit) [inline CONTRIBUTING.md:19]: Are you planning to fill this section?

File: README.md
Comment (mucahit) [inline README.md:21]: Let's also add vanilla JS version with webpack + typescript

File: README.md
Comment (mucahit) [inline README.md:39]: Let's shorten this example a bit, no need for the React details. We already have examples 

```typescript
// Connect handler
peraWallet
  .connect()
  .then((newAccounts) => {
    // Setup the disconnect event listener
    peraWallet.connector?.on("disconnect", handleDisconnectWalletClick);

    setAccountAddress(newAccounts[0]);
  })
  .reject((error) => {
    // You MUST handle the reject because once the user closes the modal, peraWallet.connect() promise will be rejected.
    // For the async/await syntax you MUST use try/catch
    if (error?.data?.type !== "CONNECT_MODAL_CLOSED") {
      // log the necessary errors
    }
  });
```

```typescript
// On the every page refresh 
peraWallet.reconnectSession().then((accounts) => {
  // Setup the disconnect event listener
  peraWallet.connector?.on("disconnect", handleDisconnectWalletClick);

  if (accounts.length) {
    setAccountAddress(accounts[0]);
  }
});
```

File: README.md
Comment (mucahit) [inline README.md:116]:
```suggestion
You can override the z-index using the `.pera-wallet-connect-modal` class so that the modal does not conflict with another component on your application.
```

File: README.md
Comment (mucahit) [inline README.md:137]: All contributions are welcomed! To get more information about the details, please read the [contribution](./CONTRIBUTING.md) guide first.

**Review by mucahit — CHANGES_REQUESTED** [review]
Thanks! @yigiterdev

File: CONTRIBUTING.md
Comment (yigiterdev) [reply CONTRIBUTING.md:19]: Actually, development is like the main title of the following titles here.

Comment (mucahit) [reply CONTRIBUTING.md:19]: oh sorry, makes sense

**Review by mucahit — APPROVED** [review]
thanks :tada:, it's looking good to me. let's just wait for the detailed docs to be published before we merge this one.

**Pull request merged**
